cmd/thanos/receive.go

```diff
@@ -173,6 +175,15 @@ func runReceive(
 		return errors.Wrapf(err, "migrate legacy storage in %v to default tenant %v", conf.dataDir, conf.defaultTenantID)
 	}
 
+	relabelContentYaml, err := conf.relabelConfigPath.Content()
```

Comment:
Very useful feature, could make it reloadable?

Reply:
Sounds good. I prefer having a separate issue and doing it in another pr.

Reply:
@hanjm Can you please open an issue for this feature?

Reply:
#5398